client/src/components/HistoryViewer/HistoryViewer.scss

```diff
@@ -10,7 +10,7 @@
 
 // Used in conjunction with .panel--padded, removes top and bottom padding
 .panel--padded-side {
-  padding-bottom: 0;
+  padding-bottom: 0px;
```

Comment:
Any reason for this and not the line below it?

Reply:
ha, I made a whoops at some point in the commit history, then undid it (but forgot to undo the units too).